src/intervals/functions.jl

```diff
@@ -4,61 +4,140 @@
 # Integer power:
 function ^{T<:Real}(a::Interval{T}, n::Integer)
```

Comment:
This is something I certainly tried, but it didn't quite worked; it was probably my implementation, but it simply was not yielding what it should (ITF1788).